src/features/account-transaction-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box } from '@stacks/ui';
+import { InfiniteTransactionsList } from '@components/infinite-item-list';
+import { Section } from '@components/section';
+import * as React from 'react';
+import { useAccountInViewTransactions } from '../hooks/currently-in-view-hooks';
```

Comment:
Should we make an `@hooks` alias?

Reply:
yep! I started moving hooks out of `@common/hooks` to `./src/hooks`, but wanted to wait until we get our two PRs merged before doing a larger move like that to avoid conflicts